include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -319,4 +319,10 @@
 
 #define _ONEDPL_BUILT_IN_STABLE_NAME_PRESENT __has_builtin(__builtin_sycl_unique_stable_name)
 
+// When compile with Intel(R) oneAPI DPC++ Compiler macro is on
+// We need this macro to use compiler specific implementation of SYCL
+#if defined(__INTEL_LLVM_COMPILER) and defined(SYCL_LANGUAGE_VERSION)
```

Comment:
there is no usage of `and` in the file or at least I don't see. Please be consistent and use `&&`

Reply:
Thank you, I have overlooked. Fixed.